WcaOnRails/app/controllers/delegates_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class DelegatesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Hmmm... it's hard for me to think of any utility functions that only make sense in a controller where *every* route requires the user to be logged in.